src/index.js

```diff
@@ -30,14 +25,8 @@ EDIT BELOW FOR PRESENTATION CHANGES
 --------------------------------------------------------------------------------------------------------------------------------
 */
 
-// the following places the camera PiP in the lower right
-//const image_location_top = 76;  // up-down after rotation
-//const image_location_left = 83; // left-right after rotation
-
 const image_location_top = 76;  // up-down after rotation
 const image_location_left = 83; // left-right after rotation
-
-//const vidPath = '/GG-vertical.mp4'
 const vidPath = '/GG-horizontal-HD.mp4'
```

Comment:
if we're not providing the .mp4 or content, this can be dynamically finding media copied to the root of sd